scripts/cbmc_json_parser.py

```diff
@@ -369,23 +370,41 @@ def has_check_failure(properties, message):
             return True
     return False
 
-
 def filter_reach_checks(properties):
     return filter_properties(properties, GlobalMessages.REACH_CHECK_DESC)
 
+def filter_sanity_checks(properties):
+    """
+    Specific filter for sanity_check checks that removed checks with SUCCESS status
+    """
+    filtered_properties = []
+    property_class = "sanity_check"
+    for property in properties:
+        if property_class in extract_property_class(property) and property["status"] == "SUCCESS":
+            pass
+        else:
+            filtered_properties.append(property)
+
+    return filtered_properties
 
-def filter_properties(properties, message):
+def filter_properties(properties, message=None, property_class=None):
```

Comment:
The changes to this function are no longer needed, right?

Reply:
Yes, redundant changes.